compiler/tests/features.rs

```diff
@@ -75,17 +75,41 @@ fn test_assert() {
 
         let exit1 = harness.capture_call(&mut executor, "bar", &[uint_token(4)]);
 
-        assert!(matches!(
-            exit1,
-            evm::Capture::Exit((evm::ExitReason::Revert(_), _))
-        ));
+        match exit1 {
+            evm::Capture::Exit((evm::ExitReason::Revert(_), output)) => assert_eq!(output.len(), 0),
```

Comment:
Without a revert string the data will simply be empty (NOT the same as having a revert string of `""`)